R/Rules-methods.R

```diff
@@ -1973,6 +2023,15 @@ setMethod("stopTrial",
       ## so can we stop?
       doStop <- prob >= stopping@prob
 
+      report_label <- if (isTRUE(stopping@report_label == "")) {
+        paste(
+          "P(MTD >", x@thresh, "* NBD | P(DLE) = ", x@target, ") \u2265 ",
```

Comment:
same here: what is the `x` here? I do not see this object on the list of explicit arguments to this function.